capstone-backend/src/main/java/com/google/sps/data/Club.java

```diff
@@ -1,30 +1,22 @@
 package com.google.sps.data;
 
-import java.util.Collection;
 import java.util.ArrayList;
-import java.util.HashSet;
+import java.util.List;
 
- 
-/**
- * Class representing a Club.
- *
- * Note: The private variables in this class are converted into JSON.
- */
-public class Club {
+
+public class Club extends BaseEntity {
```

Comment:
This seems like changes made in previous PRs, not sure why they're showing up here. Maybe you need to base this off of `master`.